tests/functional/labels.js

```diff
@@ -0,0 +1,69 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'intern',
+  'intern!object',
+  'intern/chai!assert',
+  'require'
+], function (intern, registerSuite, assert, require) {
+  'use strict';
+
+  var url = function (num) {
+    return intern.config.siteRoot + '/issues/' + num;
+  };
+
+  registerSuite({
+    name: 'labels',
+
+    setup: function () {
+      // We should be logged before starting the labels test.
+      return this.remote
+        .setFindTimeout(intern.config.wc.pageLoadTimeout)
+        .get(require.toUrl(url(100)))
+        .findByCssSelector('.js-login-link').click()
```

Comment:
Ah, one more thing is we have the `/login` and `/logout` routes we can always visit--which redirect to where you came from. So instead of clicking `.js-login-link` you could `.get(require.toUrl("/login"))` to perform a login without risking logging out. Same for /logout, etc.